### PR TITLE
records: expose theme field in search

### PR DESCRIPTION
**Pull request by kpsherva**
* add dedicated route
* closes https://github.com/zenodo/zenodo-rdm/issues/691

### CODE REVIEW

File: invenio_communities/templates/semantic-ui/invenio_communities/details/header.html [hunk 1]
```diff
@@ -104,14 +104,15 @@ <h1 class="ui medium header mb-0">{{ community.metadata.title }}</h1>
...
       </div>
-
       <div
         class="sixteen wide mobile sixteen wide tablet three wide computer right aligned middle aligned column">
-        <a href="/uploads/new?community={{ community.slug }}"
-           class="ui positive button labeled icon rel-mt-1 theme-secondary">
-          <i class="upload icon" aria-hidden="true"></i>
-          {{ _("New upload") }}
-        </a>
+        {%- if not community_use_jinja_header %}
```
Comment (zzacharo) [inline invenio_communities/templates/semantic-ui/invenio_communities/details/header.html:109]: I might miss some context but why do we show the "New Upload" button only when we don't have a branded community (`community_use_jinja_header` flag)?

**Review by slint — APPROVED** [review]
Reviewed with @0einstein0 and LGTU!

**Review by slint — APPROVED** [review]
LGTM!

File: invenio_communities/templates/semantic-ui/invenio_communities/details/header.html (hunk 1 above)
Comment (kpsherva) [reply invenio_communities/templates/semantic-ui/invenio_communities/details/header.html:109]: we dont show the new upload button when we are already in the deposit form (discussed also in person)

**Pull request merged**